### DISCUSSION TITLE
Codemagic Static Pages: SSL Error

### DISCUSSION THREAD

**Discussion by mhrst** [codemagic-ci-cd/codemagic-docs#1618] [Q&A] [answered]
I cannot access the web-app published by the static pages option due to an SSL error:

https://sb.qrl.app.codemagic.app/

```
curl: (60) SSL: no alternative certificate subject name matches target host name 'sb.qrl.app.codemagic.app'
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

Is there something I'm missing?

**Comment by dtrdic** [comment]
hey @mhrst could you share your build id with us, so we can take a look?

**Reply by dtrdic** [reply, marked as answer]
but in the meantime, you can try to edit the subdomain name so that it doesn't contain a dot (.) in its name.
this format will work; sbqrlapp.codemagic.app

**Reply by mhrst** [reply]
That worked! Thanks

**Reply by dtrdic** [reply]
Glad to hear it :) cheers

**Comment by mhrst** [comment]
Build ID: 6453051c1bd678ee0e57cef3

I will give the subdomain change a try.
